frontend/components/forms/queries/QueryForm/QueryForm.tests.jsx

```diff
@@ -76,15 +76,17 @@ describe('QueryForm - component', () => {
   it('enables the Save Changes button when the name input changes', () => {
     const form = mount(<QueryForm formData={{ ...query, query: queryText }} onTargetSelect={noop} />);
     const inputFields = form.find('InputField');
-    const nameInput = inputFields.find({ name: 'name' });
-    const saveChangesOption = form.find('li.dropdown-button__option').first().find('Button');
+    const nameInput = inputFields.find('input[name="name"]');
+    let saveChangesOption = form.find('li.dropdown-button__option').first().find('Button');
 
     expect(saveChangesOption.props()).toInclude({
       disabled: true,
     });
 
     fillInFormInput(nameInput, 'New query name');
+    nameInput.simulate('change', { target: { value: 'New query name' } });
 
+    saveChangesOption = form.find('li.dropdown-button__option').first().find('Button');
```

Comment:
Why is it necessary to reassign the same selector?

Reply:
The components returned from the selector seem to now be immutable in the new version of Enzyme... Only the root element is "rerendered", so you have to find them again from that root element in order to capture the changes.

Reply:
That seems very annoying